README.md

```diff
@@ -71,7 +71,7 @@ Channel: #refugerestrooms
 
 Copyright (C) 2014–2016 Teagan Widmer and contributors
 
-This program is free software; you can redistribute it and/or modify
+This program is free software; you can redistribute and/or modify
```

Comment:
there are two it's in the previous version, and only one is needed grammatically i think.